python/sglang/srt/openai_api/adapter.py

```diff
@@ -1417,6 +1426,38 @@ async def generate_stream_resp():
             prompt_tokens = {}
             completion_tokens = {}
             cached_tokens = {}
+            def get_usage():
```

Comment:
It feels weird to define a nested function here. 
What is the difference between this function and the one defined at L748?